### ISSUE TITLE
[feature]: Directional BiRel

### ISSUE THREAD

**Issue by oliv3r** [plantuml-stdlib/C4-PlantUML#214]
For normal relations, `Rel` we have directional hints available `Rel_L[eft]` for example. This is (often/always) used to help the renderer orient things smarter.

However, when using a BiRel relationship, we can have a two-headed arrow, but no hint to the renderer.

I thus recommend we introduce `BiRel_L[eft]`, `BiRel_R[ight]`, `BiRel_U[p]` and `BiRel_D[own]` to match our `Rel` breatheren.

> Note that we have to be careful not to break `LAYOUT_LEFT_RIGHT`or `LAYOUT_LANDSCAPE`

**Comment by kirchsth** [comment]
Hi @oliv3r,

are you using a (very) old version? BiRel supports directions like below.
Or do you mean something different?

```plantuml
@startuml
!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
HIDE_STEREOTYPE()

Person(x, "X")
System(s1, "S1")
System(s2, "S2")
System(s3, "S3")
System(s4, "S4")

BiRel_U(x, s1, "uses")
BiRel_R(x, s2, "uses")
BiRel_D(x, s3, "uses")
BiRel_L(x, s4, "uses")
@enduml
```
![](https://www.plantuml.com/plantuml/png/NSn1IyCm6CRnUpx5tAa3rx2s9q_3BQ0miRGRwAbaxOiB96d9-mRrsvl4mqgFpm-U_nv9El96imTbU-q7MDs89dpYtCdl_4lHpLyzWkj7Is0fxqV39orj_6oH1gsk_51jwqYNjnCt4WbSeEuG7b9PSFaK-o-lHz6rPz68z_DdBR8DOpKu76tsyxXQVwmthFqDLvFXBknsDuCYGZ63Ca8vWof25O0zgmPqTudH5F886FoVcyJ5ae-9ooMV4bTtteCTlD5_)
BR 
Helmut

**Comment by oliv3r** [comment]
Ah, I'm using it through gitlab, with the include you list; Funnily enough; it is actually working now, sorry for the noise!

**Comment by kirchsth** [comment]
@oliv3r,

you can check with %version() the active Plantuml version.

```plantuml
@startuml
[ %version() ]
@enduml
```

Maybe you can ask on GitLab side if they can update to a newer version (e.g. like 1.2022.2 then you would have the last version of C4-PlantUML (2.4.0) and it contains fixes of some bugs which I found in 1.2022.2 beta [which could be in older versions too])

BR Helmut

**Comment by oliv3r** [comment]
Hey Helmut, well i'm using the github includes still sadly: So I have:
```plantuml
!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Context.puml

LAYOUT_TOP_DOWN()
SHOW_PERSON_OUTLINE()
```
